vavr/generator/Generator.scala

```diff
@@ -1953,7 +1953,7 @@ def generateMainClasses(): Unit = {
               /$javadoc
                * The ${j.ordinal} element of this tuple.
                */
-              public final T$j _$j;
+              transient public T$j _$j;
```

Comment:
`transient` means that you want to ignore field during serialization - tuple elements can't be ignored

Reply:
Indeed. The transient modifier will likely break other serialization libraries, too.

Reply:
There is an implementation added in this class for readObject() and writeObject() to explicitly serialize and deserialize these fields so that they are not ignored.